tests/foreman/cli/test_repository.py

```diff
@@ -1227,9 +1226,12 @@ def test_positive_synchronize_rpm_repo_ignore_content(self):
                          'content not ignored correctly')
         self.assertEqual(repo['content-counts']['errata'], '2',
                          'content not synced correctly')
-        if not bz_bug_is_open(1664549):
-            self.assertEqual(repo['content-counts']['source-rpms'], '3',
-                             'content not synced correctly')
+
+        # BEGIN workaround for BZ: 1664549
+        # self.assertEqual(repo['content-counts']['source-rpms'], '3',
+        #                 'content not synced correctly')
+        # END workaround for BZ: 1664549
```

Comment:
Workaround that is commented out?

Reply:
When BZ is resolved then the assertion can be re-included. I added a better note about it